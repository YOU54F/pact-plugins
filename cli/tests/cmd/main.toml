bin.name = "pact-plugin-cli"
status.code = 2
