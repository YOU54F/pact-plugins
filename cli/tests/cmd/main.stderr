pact-plugin-cli [CLIVERSION]
CLI utility for Pact plugins

USAGE:
    pact-plugin-cli [OPTIONS] <SUBCOMMAND>

OPTIONS:
    -d, --debug      Enable debug level logs
    -h, --help       Print help information
    -t, --trace      Enable trace level logs
    -V, --version    Print version information
    -y, --yes        Automatically answer Yes for all prompts

SUBCOMMANDS:
    disable    Disable a plugin version
    enable     Enable a plugin version
    env        Print out the Pact plugin environment config
    help       Print this message or the help of the given subcommand(s)
    install    Install a plugin
    list       List installed plugins
    remove     Remove a plugin
